src/libraries/icubmod/bcbBattery/bcbBattery.ini

```diff
@@ -1,5 +0,0 @@
-[plugin bcbBattery]
-type device
-name bcbBattery
-library bcbBattery
-wrapper batteryWrapper
```

Comment:
I think we missed this one on the first review, or perhaps it was wrong before...
Anyway `wrapper batteryWrapper` -> `EXTRA_CONFIG WRAPPER=BatteryWrapper`

Reply:
fixed by 9e0dee0ef0753f6536f6c636a43913